.gitignore

```diff
@@ -8,10 +8,11 @@ work
 .idea
 out
 
-# eclipse project file
+# Eclipse and VSCode project files
 .settings
 .classpath
 .project
+.factorypath
```

Comment:
```suggestion
**/.factorypath
```

Reply:
I was following the existing patterns

Reply:
Maybe we create another PR for that change